packages/wdio-cli/src/config.js

```diff
@@ -148,6 +153,12 @@ export const CLI_PARAMS = [{
 }]
 
 export const QUESTIONNAIRE = [{
+    type: 'checkbox',
+    name: 'browsers',
+    message: 'On which browsers will you be testing?',
```

Comment:
This question should not come first. Depending on where the tests are being run (local or on a cloud) we can already make assumptions to what to ask. For example: if running tests locally we don't need to ask for platform and version however when running tests on SauceLabs we definitely want to do that.